src/ui.tsx

```diff
@@ -59,7 +59,7 @@ async function startUi() {
     const stateChanges = await backgroundService.getState([
       'initialized',
       'locked',
-      'assets',
+      ...('currentNetwork' in changes ? ['assets'] : []),
```

Comment:
I constantly forget why we need to get these fields explicitly always. Guess we need an explanation in a code comment :-)